src/overlays/actors/ovl_Obj_Um/z_obj_um.c

```diff
@@ -1,39 +1,66 @@
 /*
  * File: z_obj_um.c
  * Overlay: ovl_Obj_Um
- * Description: Cremia's cart and milk run quest
+ * Description: Cremia's Cart and milk run minigame
  */
 
 #include "z_obj_um.h"
+#include "overlays/actors/ovl_En_In/z_en_in.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
 
 #define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8 | ACTOR_FLAG_10 | ACTOR_FLAG_20)
 
 #define THIS ((ObjUm*)thisx)
 
+/**
+ * weekEventReg flags checked by this actor:
+ * - gSaveContext.save.weekEventReg[22] & 1: Aliens defeated
+ *     If false: The actor doesn't spawn
+ * - gSaveContext.save.weekEventReg[31] & 0x40
+ *     If true: Cremia doesn't explain again she'll deliever milk to town
+ * - gSaveContext.save.weekEventReg[31] & 0x80
+ *     If true: Triggers cutscene on Romani's Ranch
+ * - gSaveContext.save.weekEventReg[34] & 0x80
+ *     If true: Doesn't spawn on Romani's Ranch
+ * - gSaveContext.save.weekEventReg[52] & 1
+ *     If true: Doesn't spawn on Romani's Ranch or Milk Road
+ * - gSaveContext.save.weekEventReg[52] & 2
+ *     If true: Doesn't spawn on Romani's Ranch or Milk Road
+ * - gSaveContext.save.weekEventReg[59] & 2
+ *     If true: Doesn't spawn again on Milk Road
+ *
+ * weekEventReg flags set by this actor:
+ * - gSaveContext.save.weekEventReg[31] |= 0x40: Cremia offered a ride
+ *     Cremia offered a ride accross the Milk Road to Player
+ * - gSaveContext.save.weekEventReg[31] |= 0x80: Player is in Milk Run
+ *     Player accepts the ride and is with Cremia during the Milk Run.
+ * - gSaveContext.save.weekEventReg[34] |= 0x80: Cremia does Milk Run alone
+ *     Player didn't interact or didn't accept the ride
+ * - gSaveContext.save.weekEventReg[52] |= 1: Won Milk Run minigame
+ *     At least one pot is safe. Turns off the "Lose Milk Run minigame"
+ * - gSaveContext.save.weekEventReg[52] |= 2: Lose Milk Run minigame
+ *     Every pot was broken by bandits. Turns off the ""Win" Milk Run minigame"
```

Comment:
Is there a reason that "Win" is in quotes here?

Reply:
Doesn't seem like... I'll remove them